README.md

```diff
@@ -1,13 +1,13 @@
-# LBRY Android
+# LBRY F-Droid
 [![pipeline status](https://ci.lbry.tech/lbry/lbry-android/badges/master/pipeline.svg)](https://ci.lbry.tech/lbry/lbry-android/commits/master)
 [![GitHub license](https://img.shields.io/github/license/lbryio/lbry-android)](https://github.com/lbryio/lbry-android/blob/master/LICENSE)
 
 An Android browser and wallet for the [LBRY](https://lbry.com) network.
 
+<a href="https://f-droid.org/packages/io.lbry.browser"><img src="https://raw.githubusercontent.com/f-droid/artwork/master/badge/get-it-on-en-us.png" title="Get LBRY on F-droid"></a>
```

Comment:
Remove this change.

This is the source code of the app. Final users can read the instructions on the Installation section.

Also, lbry-android doesn't provide something like this on their Readme file.